cilium/CFP-25243-high-scale-ipcache.md

```diff
@@ -0,0 +1,116 @@
+# CFP-25243: High-Scale IPcache
+
+**SIG: SIG-DATAPATH**
+
+**Begin Design Discussion:** 2023-04-26
+
+**Cilium Release:** 1.14
+
+**Authors:** Paul Chaignon <paul@isovalent.com>
+
```

Comment:
Would it be fair to say that this proposal is now Dormant? I understand that some aspects were accepted and implemented but not all. That said, we could merge it in the current state even if there are open questions.

```suggestion

**Status:** Dormant
```

(What does this mean? See https://github.com/cilium/design-cfps/pull/42 for a proposal)